qc/find_holes.py

```diff
@@ -199,7 +199,9 @@ def main(args):
 
     # should be as a terrain grid - but problems with high veg on fields!!!
     pc = pointcloud.fromAny(pargs.las_file).cut_to_class(cut_to)
-    pc_ref = pointcloud.fromAny(pargs.ref_data).cut_to_class(5)
+    
+    # "class" is a reserved keyword...
+    pc_ref = pointcloud.fromAny(pargs.ref_data).cut_to_class(getattr(pargs, 'class'))

```

Comment:
`pargs.class` is possible here and the same pattern is used throughout the code base. It is simpler to read and write than `getattr(pargs, 'class'))`.

Reply:
Tried using pargs.class. It causes a SyntaxError to be raised.

Reply:
Going with the ```dest=``` option to avoid breaking the backward compatibility of configurations for now.